libethdrivers/src/plat/imx6/imx6.c

```diff
@@ -307,7 +317,7 @@ static int setup_desc_ring(imx6_eth_driver_t *dev, ring_ctx_t *ring)
                   sizeof(void *),
                   (void **) &ring->cookies);
     if (ret) {
-        LOG_ERROR("Failed to malloc, code %d", ret);
+        ZF_LOGE("Failed to malloc, code %d", ret);
```

Comment:
Note to self: this should have been fixed in an earlier cleanup commit already, there should not be an change in this commit for the logging.

Reply:
fixed